noaa_semantic_search/test_procedures/semantic-search-common/hybrid-search-complex-queries.json

```diff
@@ -0,0 +1,70 @@
+    {
+        "operation": "hybrid-query-only-range",
+        "warmup-iterations": {{ warmup_iterations | default(25) | tojson }},
+        "iterations": {{ test_iterations | default(100) | tojson }},
+        "target-throughput": {{ target_throughput | default(2) | tojson }},
```

Comment:
As we have two modes of running the benchmarks. Lets keep default to 0. So that it first run the benchmarking mode. If the user provides a custom value then it will be picked up from params json file.

Reply:
ack